components/bots/environment.ts

```diff
@@ -19,7 +19,16 @@ export default class Environment {
             }
 
             const name = await this.askPredefinedEnvironmentName();
-            
+
+            const existEnvs = await this.listEnvironment();
+            for (const existEnv of existEnvs) {
+                if (existEnv.name === name) {
```

Comment:
please normalize both environment name before comparing it

Reply:
resolved in latest commit on this branch